nailgun/entities.py

```diff
@@ -3702,6 +3708,89 @@ def rebuild_config(self, synchronous=True, timeout=None, **kwargs):
+            hostgroup.assign_ansible_roles(data={'ansible_role_ids':
+            [ansible_role_id1, ansible_role_id2]})
+
+        :param synchronous: What should happen if the server returns an HTTP
+            202 (accepted) status code? Wait for the task to complete if
+            ``True``. Immediately return the server's response otherwise.
+        :param timeout: Maximum number of seconds to wait until timing out.
+            Defaults to ``nailgun.entity_mixins.TASK_TIMEOUT``.
+        :param kwargs: Arguments to pass to requests.
+        :returns: The server's response, with all JSON decoded.
+        :raises: ``requests.exceptions.HTTPError`` If the server responds with
+            an HTTP 4XX or 5XX message.
+
+        """
+        kwargs = kwargs.copy()
+        kwargs.update(self._server_config.get_client_kwargs())
+        response = client.post(self.path('assign_ansible_roles'), **kwargs)
+        return _handle_response(response, self._server_config, synchronous, timeout)
+
+    def list_ansible_roles(self, synchronous=True, timeout=None, **kwargs):
+        """List all Ansible Roles assigned to a hostgroup
+
+        :param synchronous: What should happen if the server returns an HTTP
+            202 (accepted) status code? Wait for the task to complete if
+            ``True``. Immediately return the server's response otherwise.
+        :param timeout: Maximum number of seconds to wait until timing out.
+            Defaults to ``nailgun.entity_mixins.TASK_TIMEOUT``.
+        :param kwargs: Arguments to pass to requests.
+        :returns: The server's response, with all JSON decoded.
+        :raises: ``requests.exceptions.HTTPError`` If the server responds with
+            an HTTP 4XX or 5XX message.
+
+        """
+        kwargs = kwargs.copy()
+        kwargs.update(self._server_config.get_client_kwargs())
+        response = client.get(self.path('ansible_roles'), **kwargs)
+        return _handle_response(response, self._server_config, synchronous, timeout)
+
+    def add_ansible_role(self, synchronous=True, timeout=None, **kwargs):
+        """Add single Ansible Role to a hostgroup
+
+        :param synchronous: What should happen if the server returns an HTTP
+            202 (accepted) status code? Wait for the task to complete if
+            ``True``. Immediately return the server's response otherwise.
+        :param timeout: Maximum number of seconds to wait until timing out.
+            Defaults to ``nailgun.entity_mixins.TASK_TIMEOUT``.
+        :param kwargs: Arguments to pass to requests.
+        :returns: The server's response, with all JSON decoded.
+        :raises: ``requests.exceptions.HTTPError`` If the server responds with
+            an HTTP 4XX or 5XX message.
+        """
+        kwargs = kwargs.copy()
+        kwargs.update(self._server_config.get_client_kwargs())
+        #path = f'{self.path("ansible_roles")}/{kwargs["data"].pop("ansible_role_id")}'
```

Comment:
```suggestion
```

Reply:
The fix for this path is not yet in Snap, will uncomment it when the fix will be delivered to us

Reply:
Oh ok, so is this not functional at the moment?

Reply:
the path below this line is functional and a commented path will be functional after the fix

Reply:
OK would you like to merge as is and update when the snap is in?

Reply:
Updated, tested locally with snap 1 it is fixed, and in upcoming stream snap it should be there as well, so we're good to merge it now